penny_university_frontend/src/components/modal/confirmation/deleteFollowUp.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import { ThunkDispatch } from 'redux-thunk'
+import { AnyAction } from 'redux'
+import {
+  ModalHeader, ModalBody, Button, ModalFooter,
+} from 'reactstrap'
+import modalDispatch from '../dispatch'
+import { deleteFollowUp } from '../../../actions/chat';
+
+type DeleteFollowUpModalProps = {
+  deleteFollowUp: (followUpID: number, chatID: number) => AnyAction,
+  followUpID: number,
+  chatID: number,
+}
+
+const DeleteFollowUpModal = ({ deleteFollowUp, followUpID, chatID }: DeleteFollowUpModalProps) => {
+  const deleteOnPress = () => {
+    deleteFollowUp(followUpID, chatID)
+    modalDispatch.close()
+  }
+  return (
+    <>
+      <ModalHeader toggle={modalDispatch.close}>Delete Follow Up</ModalHeader>
+      <ModalBody>
+        Are you sure you want to delete this Follow Up? This action cannot be undone.
```

Comment:
nit: We might want to think about putting our strings in a file in the event we want to add translations later